cosmovisor/README.md

```diff
@@ -30,6 +30,28 @@ To install the latest version of `cosmovisor`, run the following command:
 go install github.com/cosmos/cosmos-sdk/cosmovisor/cmd/cosmovisor@latest
 ```
 
+To install a previous version, you can specify the version. IMPORTANT: Chains that use Cosmos-SDK v0.42.x MUST use Cosmovisor v0.1.0
```

Comment:
```suggestion
To install a previous version, you can specify the version. IMPORTANT: Chains that use Cosmos-SDK v0.42.x and want to use auto-download feature MUST use Cosmovisor v0.1.0
```

Reply:
we recommend using v1.0 in all cases except for a auto-download upgrade from 0.42 based chain.